templates/accounts/account/email.html

```diff
@@ -1,7 +1,6 @@
 {% extends "account/base.html" %}
 
 {% load i18n %}
-{% load url from future %}
```

Comment:
I'm going to have to view each of them one at a time, but so far, there are no changes from the app. Which is to say, in comparing this version of allauth's templates to the ones we copied from (presumably) 0.19, there have been no changes which are not ones we ourselves made for our own purposes (eg. to add in CSS links, comment out sections we don't use).